contracts/mocks/PausableMock.sol

```diff
@@ -14,7 +14,9 @@ contract PausableMock is Pausable {
     }
 
     function normalProcess() external whenNotPaused {
-        count++;
+        unchecked{
+            count++;
+        }
```

Comment:
we don't care about the gas efficiency of mocks. Lets leave them as they are for the sake of readability.